src/controller/datacontroller.js

```diff
@@ -113,6 +113,21 @@ export default class DataController {
 	 * @returns {String} Output data.
 	 */
 	get( rootName = 'main' ) {
+		if ( !_checkIfRootsExists( [ rootName ], this.model.document.getRootNames() ) ) {
```

Comment:
```suggestion
		if ( !this._checkIfRootsExists( [ rootName ] ) ) {
```

Reply:
Moved `_checkIfRootsExists` function to class.